WS2012R2/lisa/remote-scripts/ica/Core_LIS_CD.sh

```diff
@@ -97,7 +97,14 @@ if [ $os_VENDOR != "Ubuntu" ] && [ $os_VENDOR != "Debian" ]; then
 fi
 sleep 1
 LogMsg "Mount the CDROM"
-mount /dev/cdrom /mnt/
+for drive in $(ls /dev/cdrom*)
+do
+    blkid $drive
```

Comment:
You should make sure /mnt/ is not mounted.
In case you have more than 1 CDs attached to the VM, the CD to be mounted will not be the one used for the test. The test will use the first CD found, since the other attempts to mount in /mnt/ will fail.